src/main/cpp/subsystems/DooHickey.cpp

```diff
@@ -10,22 +10,23 @@
 
 namespace
 {
-  constexpr double kF{0.3};
+  constexpr double kF{0.0};
   constexpr double kP{0.1};
   constexpr double kI{0.0};
   constexpr double kD{0.0};
+  constexpr double kNumberOfRotations{4};
+  constexpr double kTicksPerRotation{2048};
+  constexpr double radiusCW{16}; //16" radius of Control panel 
+  constexpr double radiusDW{1.5};  //1.5" radius of DooHickey wheel (3" diameter)
+  constexpr double TargetPos = (kNumberOfRotations*(radiusCW/radiusDW)) * kTicksPerRotation;
   constexpr frc::DoubleSolenoid::Value kHickeyEngage {frc::DoubleSolenoid::kForward};
   constexpr frc::DoubleSolenoid::Value kHickeyDisengage {frc::DoubleSolenoid::kReverse};
 }
 
-DooHickey::DooHickey(
-    const wpi::Twine& name,
-    WPI_TalonFX& spinner,
-    frc::DoubleSolenoid& UpPushyThang)
-  :mMotorSpeed{0}
+DooHickey::DooHickey(const wpi::Twine& name, Components& components)
+: mComponents(components)
+  ,mMotorSpeed{0}
   ,mTargetPos{0}
```

Comment:
Just like `mMotorSpeed`, `mTargetPos` can be a local instead of a member variable.

Reply:
After practice